website/static/js/share/stats.js

```diff
@@ -62,6 +62,9 @@ function timeGraph (data) {
         },
         legend: {
             show: false
+        },
+        tooltip: {
+          grouped: false
```

Comment:
Is there a way you can write a custom filter for what will display in the legend?